tests/test_results_table.py

```diff
@@ -1,11 +1,74 @@
 import os
```

Comment:
Looks good, was able to merge it with the tests in other pull request

Reply:
Perfect!